src/gluonts/transform/feature.py

```diff
@@ -562,7 +562,9 @@ class CountTrailingZeros(SimpleTransformation):
     Add the number of 'trailing' zeros in each univariate time series as a feature, to be
     used when dealing with sparse (intermittent) time series. For
     example, for a time series `[0, 0, 2, 3, 0]`, the number of trailing
-    zeros will be 1.
+    zeros will be 1. If an n-dimensional array is provided, the first 1-d array along the first
+    dimension. For example, for a 2-d array A, `A[:, 0]` (the first column) and for a 3-d array,
+    `A[:, 0, 0]`.
```

Comment:
Is there a specific reason for this behavior? The "cleanest" way I would imagine such a transformation to work on multidimensional arrays is: it has an `axis` parameter, and it counts trailing zeros along such axis; an additional flag could indicate whether the axis is to be removed from the output, or should stay and have length 1.

That would be very "numpythonic" I think.

Reply:
Mm.. I'll check this.